diff --git a/node_modules/react-native/scripts/react_native_pods.rb b/node_modules/react-native/scripts/react_native_pods.rb
index e9f5a89..7d32a66 100644
--- a/node_modules/react-native/scripts/react_native_pods.rb
+++ b/node_modules/react-native/scripts/react_native_pods.rb
@@ -46,7 +46,7 @@ end
 def prepare_react_native_project!
   # Temporary solution to suppress duplicated GUID error & master specs repo warning.
   # Can be removed once we move to generate files outside pod install.
-  install! 'cocoapods', :deterministic_uuids => false, :warn_for_unused_master_specs_repo => false
+  install! 'cocoapods', :deterministic_uuids => false
 
   ReactNativePodsUtils.create_xcode_env_if_missing
 end
